Extend/Language.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ *
+ */
+
+namespace Wirecard\Oxid\Extend;
+
+/**
+ * Extends the OXID Language class
+ *
+ * @mixin \OxidEsales\Eshop\Core\Language
+ *
+ * @since 1.0.1
+ */
+class Language extends Language_parent
+{
+    const TRANSLATION_KEY_PREFIX = 'wdpg_';
+    const FALLBACK_LANGUAGE_ABBR = 'en';
+
+    /**
+     * @var int
+     *
+     * @since 1.0.1
+     */
+    private $_iFallbackId = null;
+
+    /**
+     * @inheritdoc
+     *
+     * Provides a fallback language string if the chosen language is not available for the plugin
+     *
+     * @param string $sStringToTranslate Initial string
+     * @param int    $iLang              optional language number
+     * @param bool   $blAdminMode        force to load language constant from admin/shops language file
+     *
+     * @return string
+     *
+     * @since 1.0.1
+     */
+    public function translateString($sStringToTranslate, $iLang = null, $blAdminMode = null)
+    {
+        $sParentString = parent::translateString($sStringToTranslate, $iLang, $blAdminMode);
+
+        if (!$this->isTranslated() && strpos($sStringToTranslate, self::TRANSLATION_KEY_PREFIX) === 0) {
+            return $this->_getFallbackString($sStringToTranslate, $blAdminMode);
+        }
+
+        return $sParentString;
+    }
+
+    /**
+     * Returns the fallback language's string for the provided key
+     *
+     * @param string $sStringToTranslate
+     * @param bool   $blAdminMode
```

Comment:
Please, rename "blAdminMode" to `bAdminMode` in order to stay consistent within our plugin.